supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb

```diff
@@ -121,7 +121,7 @@
         "# Set the model name from Hugging Face and task type\n",
         "# sentence-transformers model\n",
         "hf_model_id='sentence-transformers/all-mpnet-base-v2'\n",
-        "tm = TransformerModel(hf_model_id, \"text_embedding\")\n",
+        "tm = TransformerModel(model_id=hf_model_id, task_type="text_embedding")\n",
```

Comment:
This one too, the quotes need to be escaped.

Reply:
updated. thank you.